.travis.yml

```diff
@@ -3,17 +3,9 @@ dist: latest
 
 python:
   - 3.5
+  - 3.8
 
 env:
-  - TOXENV=django111-drf39
-  - TOXENV=django111-drf310
-  - TOXENV=django111-drflatest
-  - TOXENV=django20-drf39
-  - TOXENV=django20-drf310
-  - TOXENV=django20-drflatest
-  - TOXENV=django21-drf39
-  - TOXENV=django21-drf310
-  - TOXENV=django21-drflatest
```

Comment:
You can also move the `quality` and `docs` envs from `matrix` to `envs` and remove the `matrix` section altogether.

Reply:
@UsamaSadiq IMO there is no need to run quality and docs test on python 3.5 as well as on python 3.8